bin/plotting/pycbc_faithsim_plots

```diff
@@ -0,0 +1,208 @@
+
+    if vmin is not None:
+        vmin = float(vmin)
+
+    if vmax is not None:
+        vmax = float(vmax)
+
+    fig = pylab.figure(num=None)
+    pylab.scatter(
+        xval,
+        yval,
+        c=cval,
+        linewidths=0,
+        s=3,
+        vmin=vmin,
+        vmax=vmax,
+        cmap=cmap,
+        alpha=0.7,
+    )
+
+    if len(neg_idx) > 0:
+        pylab.scatter(
+            xval[neg_idx],
+            yval[neg_idx],
+            c="red",
+            marker="x",
+            label="Unable to generate waveforms",
+        )
+        matplotlib.pyplot.legend()
+
+    if cval is not None:
+        bar = pylab.colorbar()
+        bar.set_label(cname)
+
+    pylab.xlabel(xname)
+    pylab.ylabel(yname)
+
+    if xmin is None:
+        xmin = min(xval)
+    else:
+        xmin = float(xmin)
+
+    if ymin is None:
+        ymin = min(yval)
+    else:
+        ymin = float(ymin)
+
+    pylab.xlim(xmin, max(xval))
+    pylab.ylim(ymin, max(yval))
+
+    ax = fig.gca()
+    if majorL:
+        majorL = float(majorL)
+        ax.xaxis.set_major_locator(MultipleLocator(majorL))
+        ax.yaxis.set_major_locator(MultipleLocator(majorL))
+    if minorL:
+        minorL = float(minorL)
+        ax.xaxis.set_minor_locator(MultipleLocator(minorL))
+        ax.yaxis.set_minor_locator(MultipleLocator(minorL))
+
+    pylab.grid()
+    pylab.title(title)
+
+    pylab.savefig(out_name, dpi=500)
+
+
+parser = argparse.ArgumentParser(description=__doc__)
+parser.add_argument(
+    "--input-file",
+    required=True,
+    help="resulting .dat file from pycbc_faithsim_collect_full_results",
+)
+parser.add_argument("--parameter-x", required=True)
+parser.add_argument("--parameter-y", required=True)
+parser.add_argument("--parameter-z", required=True)
+parser.add_argument("--colormap", default="jet")
+parser.add_argument("--vmin", default=None, help="min value of the colorbar")
+parser.add_argument("--vmax", default=None, help="max value of the colorbar")
+parser.add_argument("--xmin", default=None)
+parser.add_argument("--ymin", default=None)
+parser.add_argument("--majorL", default=None)
+parser.add_argument("--minorL", default=None)
+parser.add_argument("--output-plot", required=True, help="name of the output plot")
+
+args = parser.parse_args()
+
+derived_func_map = {
+    "total_mass": lambda d: mtotal_from_mass1_mass2(d["mass1"], d["mass2"]),
+    "mass_ratio": lambda d: q_from_mass1_mass2(d["mass1"], d["mass2"]),
+    "mchirp": lambda d: mchirp_from_mass1_mass2(d["mass1"], d["mass2"]),
+    "spin1_magnitude": lambda d: (
+        d["spin1x"] ** 2 + d["spin1y"] ** 2 + d["spin1z"] ** 2
+    )
+    ** 0.5,
+    "spin2_magnitude": lambda d: (d["spin2x"] ** 2 + d["spin2y"] ** 2 + d["spin2z"] ** 2)
+    ** 0.5,
+    "eta": lambda d: eta_from_mass1_mass2(d["mass1"], d["mass2"]),
+    "chi_eff": lambda d: chi_eff(d["mass1"], d["mass2"], d["spin1z"], d["spin2z"]),
+    "s81": lambda d: d["sigma1"] / 8,
+    "s82": lambda d: d["sigma2"] / 8,
```

Comment:
s81 and s82 seem odd names here! This is the horizon distance, right (not sure what the difference is between sigma1 and sigma2, I guess the horizon distance for each of the two things being compared, which we hope are very similar).

Reply:
I guess the (sigma)^2 are the normalization factors in the match computation (from [pycbc.filter](https://pycbc.org/pycbc/latest/html/pycbc.filter.html#pycbc.filter.matchedfilter.sigma). Not sure to understand the s81 and s82, there were in the old glue code so I kept them.

Reply:
I can remove them, since there are not used here. I kept them in case someone wants to plot these parameters

Reply:
I would keep them, but can they be named more informatively? I would use "horizon_distance_1" and "horizon_distance_2"